index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/nav-bar.css">
+  <link rel="stylesheet" href="./styles/styles.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+  <title>Global Training Courses - Home</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#topics">Topics</a></li>
+          <li><a href="#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="nav-desktop">
+      <ul>
+        <li><a href="about.html">About Us</a></li>
+        <li><a href="#topics">Topics</a></li>
+        <li><a href="#courses">Courses</a></li>
+        <li><a href="#">Log In</a></li>
+        <li><a href="#">Sign Up</a></li>
+      </ul>
+    </nav>
+    <div class="title-container">
+      <h1>GLOBAL TRAINING COURSES</h1>
+      <p>A swiftly evolving business landscape requires an equally dynamic workforce. One-dimensional skill sets are no longer the norm, and not an option. To possess a blend of people, process and technology know-how makes a complete professional.
+      <br><br>Give your career a boost by mastering people and process expertise in areas like communication, critical thinking, finance, leadership, negotiation and problem-solving.</p>
+    </div>
```

Comment:
this section is the headline it would be better to be outside the header and you should stick to the design but you can customize it to your own, right now you design is not the same as the provided 

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164913945-37834144-8394-4b7c-977e-ca4191dd44fd.png)

- [ ] Design provided 
![image](https://user-images.githubusercontent.com/61790587/164913956-63a5aaac-42b6-4e05-9846-62aaad8fc786.png)

Reply:
I don't have a way to get the same backgrounds